docs/pages/reference/mesheryctl/mesheryctl-app-list.md

```diff
@@ -48,7 +48,7 @@ mesheryctl app list
 
 <pre class='codeblock-pre'>
 <div class='codeblock'>
-      --config string   path to config file (default "/home/runner/.meshery/config.yaml")
+      --config string   path to config file (default "/home/admin-pc/.meshery/config.yaml")
```

Comment:
This seems inappropriate. Is `admin-pc` a GitHub-hosted environment user?

Reply:
The rest of the PR looks great.

Reply:
`admin-pc` is my WSL instance's username. It's replaced with the GitHub one as the `doc.go` is made to run. Once the action is made to run after PR merge, it'll automatically replace. Still, I can manually replace it in the files too